Makefile

```diff
@@ -130,4 +130,4 @@ release: tar
 	@make clean-tar
 
 vendor-ansible:
-	git subtree pull --prefix ansible https://github.com/contiv/ansible HEAD --squash
```

Comment:
@erikh, you might not be expecting a comment on this but this just caught my eye and I wasn't sure if this is intentional so thought will check :).

I think  `--squash` flag might be useful to have to avoid the main projects history from being polluted by the vendored projects history?